### PR TITLE
Add support for multiline text

### PR DESCRIPTION
**Pull request by ArthurVimond**

### CODE REVIEW

**Comment by gpeal** [comment]
**Snapshot Tests**
**28**: [Report](https://happo.io/a/27/report/cab810f7db2069cd8c9f88cbe891ddb3dfc52aa9-android28) [Diff](https://happo.io/a/27/p/27/compare/master-android28/cab810f7db2069cd8c9f88cbe891ddb3dfc52aa9-android28)

**Comment by ArthurVimond** [comment]
@gpeal Actually, there are two commits for this PR: one for text drawing with glyphs, one for text drawing with fonts.

File: lottie/src/main/java/com/airbnb/lottie/model/layer/TextLayer.java [hunk 1]
```diff
@@ -137,10 +138,39 @@ private void drawTextGlyphs(
...
+
+    // Split full text in multiple lines
+    List<String> textLines = getTextLines(text);
+    int textLineCount = textLines.size();
+    for (int l = 0; l < textLineCount; l++) {
+
+      String textLine = textLines.get(l);
+      float textLineWidth = getTextLineWidthForGlyphs(textLine, font, fontScale, parentScale);
+
+			canvas.save();
```
Comment (gpeal) [inline lottie/src/main/java/com/airbnb/lottie/model/layer/TextLayer.java]: There is something funky going on with the formatting here.

Comment (ArthurVimond) [reply lottie/src/main/java/com/airbnb/lottie/model/layer/TextLayer.java]: Thanks for spotting this formatting issue from my side.

**Comment by gpeal** [comment]
**Snapshot Tests**
**28**: [Report](https://happo.io/a/27/report/f0de8de84aa35eca78a610ddbb5e01930321a3a5-android28) [Diff](https://happo.io/a/27/p/27/compare/master-android28/f0de8de84aa35eca78a610ddbb5e01930321a3a5-android28)

**Pull request merged**